newdle/client/src/components/creation/timeslots/Timeline.js

```diff
@@ -164,6 +165,7 @@ function TimelineInput({minHour, maxHour}) {
   const date = useSelector(getCreationCalendarActiveDate);
   const candidates = useSelector(getTimeslotsForActiveDate);
   const pastCandidates = useSelector(getPreviousDayTimeslots);
+  const availability = useSelector(getParticipantAvailability);
   const [editing, setEditing] = useState(!!candidates.length);
```

Comment:
Sometimes this component renders before the data is fully loaded, in which case editing may default to false.

This change, while not super pretty, avoids it:

```suggestion
  const [_editing, setEditing] = useState(false);
  const editing = _editing || !!candidates.length;
```

Reply:
How did you manage to reproduce it?

Reply:
What do you think about an early `return <Loader active />`?

Reply:
I think it makes sense, but doesn't fix the problem: you cannot return before calling all hooks, and once the state got the default value, it's gonna stay that way...

Reply:
Right, I also just realised I have to means of checking the intermediate loading state that works for creation and editing.
I'm going with this, way better than adding a new side-effect (useEffect) just for this.